managers.py

```diff
@@ -0,0 +1,47 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self):
+        self._connection = sqlite3.connect("cinema.sqlite")
+        self.table_name = "authors"
```

Comment:
The `table_name` should be `actors` instead of `authors` to match the expected database table for the `Actor` model.